### PR TITLE
Adding the KPIDemo feature into MVG

### PR DESCRIPTION
**Pull request by SergioMDCB**
This PR adds the KPIDemo feature into MVG

### CODE REVIEW

File: mvg/features/kpidemo.py [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+
+        super().summary()
+        print()
+        tab = self._results_df.describe()
+        print(tabulate(tab, headers="keys", tablefmt="psql"))
+        return tab
+
+    def plot(
+        self, kpi="rms", interactive=True, time_format=None
+    ):  # pylint: disable=W0221
```
Comment (tuix) [inline mvg/features/kpidemo.py:47]: We are using the string representation of the warnings in MVG, more readable
```suggestion
    ):  # pylint: disable=arguments-differ
```

**Review by tuix — CHANGES_REQUESTED** [review]
It appears that there are no tests for `request_analysis()` in any of the test files. We read the results from a json file and test the parser, but there is no test to ensure that the call to `request_analysis`  would have been successful or the returned results would have had the expected format. Maybe we should start adding such tests for a feature like this and add stories for other features.

File: tests/test_api_analyses.py [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+import pandas as pd
+from requests import HTTPError
+import pytest
+from mvg import MVG
+
+VALID_TOKEN = os.environ["TEST_TOKEN"]
+
+# Test data and session setup
+REF_DB_PATH = Path.cwd() / "tests" / "test_data" / "mini_charlie"
+SOURCE_ID_WAVEFORM = uuid.uuid1().hex  # generate a unique source per testrun
```
Comment (tuix) [inline tests/test_api_analyses.py:23]: I think some of these variables are unused in this file
```suggestion
# Test data setup
REF_DB_PATH = Path.cwd() / "tests" / "test_data" / "mini_charlie"
```

File: tests/test_api_analyses.py [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+
+
+@pytest.fixture()
+def waveform_source_with_measurements(session, waveform_source):
```
Comment (tuix) [inline tests/test_api_analyses.py:27]: This function can also be moved to `conftest.py` and later reused when testing other features

File: tests/test_api_analyses.py [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+            timestamp=ts_m,
+            data=accs,
+            meta=meta_info,
+            exist_ok=True,
+        )
+
+    yield waveform_source
+
+
+def test_kpidemo_analysis(session, waveform_source_with_measurements):
```
Comment (tuix) [inline tests/test_api_analyses.py:82]: don't we test any numerical values from the results?

File: tests/test_api_analyses.py (hunk 3 above)
Comment (SergioMDCB) [reply tests/test_api_analyses.py]: I had moved this fixture to conftest

File: tests/test_api_analyses.py (hunk 4 above)
Comment (SergioMDCB) [reply tests/test_api_analyses.py:23]: Not testing actual numeric values, just the format of the expected results. I had assumed that consistency on numeric results was already evaluated in vibium-cloud.

**Review by tuix — APPROVED** [review]

**Pull request merged**